src/tests/hack-status.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import HackStatus from "../components/subcomponents/hack-status";
+
+test("Status component render with props", () => {
+  const { container } = render(<HackStatus status={[0]} />);
+  expect(container.firstChild).toBeInTheDocument();
+});
+
+test("Status component render without props", () => {
+  const { container } = render(<HackStatus />);
+  expect(container.firstChild).toBeNull();
+});
+
+test("Status component rerender", () => {
+  let status = []
+  const { container, rerender } = render(<HackStatus status={status}/>);
+  expect(container.firstChild).toBeNull();
+  status = "New Hack"
```

Comment:
I'm confused, is status meant to be an array or a string? or both?

you either don't have enough tests, or they are not named well enough for the clarity...

Reply:
Oh right, it was an array and then it changed to a string! I'll fix it up